libavcodec/x86/vvc_itx_1d.asm

```diff
@@ -0,0 +1,1142 @@
+    m23_1, m23_2, m23_3, m23_4, m23_5, m23_6, m23_7, m23_8, m23_9, m23_10, m23_11, m23_12, m23_13, m23_14, m23_15, m23_16, m23_17, m23_18, m23_19, m23_20, m23_21, m23_22, m23_23, m23_24, m23_25, m23_26, m23_27, m23_28, m23_29, m23_30, m23_31, m23_32, \
+    m22_1, m22_2, m22_3, m22_4, m22_5, m22_6, m22_7, m22_8, m22_9, m22_10, m22_11, m22_12, m22_13, m22_14, m22_15, m22_16, m22_17, m22_18, m22_19, m22_20, m22_21, m22_22, m22_23, m22_24, m22_25, m22_26, m22_27, m22_28, m22_29, m22_30, m22_31, m22_32, \
+    m21_1, m21_2, m21_3, m21_4, m21_5, m21_6, m21_7, m21_8, m21_9, m21_10, m21_11, m21_12, m21_13, m21_14, m21_15, m21_16, m21_17, m21_18, m21_19, m21_20, m21_21, m21_22, m21_23, m21_24, m21_25, m21_26, m21_27, m21_28, m21_29, m21_30, m21_31, m21_32, \
+    m25_1, m25_2, m25_3, m25_4, m25_5, m25_6, m25_7, m25_8, m25_9, m25_10, m25_11, m25_12, m25_13, m25_14, m25_15, m25_16, m25_17, m25_18, m25_19, m25_20, m25_21, m25_22, m25_23, m25_24, m25_25, m25_26, m25_27, m25_28, m25_29, m25_30, m25_31, m25_32, \
+    m26_1, m26_2, m26_3, m26_4, m26_5, m26_6, m26_7, m26_8, m26_9, m26_10, m26_11, m26_12, m26_13, m26_14, m26_15, m26_16, m26_17, m26_18, m26_19, m26_20, m26_21, m26_22, m26_23, m26_24, m26_25, m26_26, m26_27, m26_28, m26_29, m26_30, m26_31, m26_32, \
+    m27_1, m27_2, m27_3, m27_4, m27_5, m27_6, m27_7, m27_8, m27_9, m27_10, m27_11, m27_12, m27_13, m27_14, m27_15, m27_16, m27_17, m27_18, m27_19, m27_20, m27_21, m27_22, m27_23, m27_24, m27_25, m27_26, m27_27, m27_28, m27_29, m27_30, m27_31, m27_32, \
+    m28_1, m28_2, m28_3, m28_4, m28_5, m28_6, m28_7, m28_8, m28_9, m28_10, m28_11, m28_12, m28_13, m28_14, m28_15, m28_16, m28_17, m28_18, m28_19, m28_20, m28_21, m28_22, m28_23, m28_24, m28_25, m28_26, m28_27, m28_28, m28_29, m28_30, m28_31, m28_32, \
+    m32_1, m32_2, m32_3, m32_4, m32_5, m32_6, m32_7, m32_8, m32_9, m32_10, m32_11, m32_12, m32_13, m32_14, m32_15, m32_16, m32_17, m32_18, m32_19, m32_20, m32_21, m32_22, m32_23, m32_24, m32_25, m32_26, m32_27, m32_28, m32_29, m32_30, m32_31, m32_32, \
+    m31_1, m31_2, m31_3, m31_4, m31_5, m31_6, m31_7, m31_8, m31_9, m31_10, m31_11, m31_12, m31_13, m31_14, m31_15, m31_16, m31_17, m31_18, m31_19, m31_20, m31_21, m31_22, m31_23, m31_24, m31_25, m31_26, m31_27, m31_28, m31_29, m31_30, m31_31, m31_32, \
+    m30_1, m30_2, m30_3, m30_4, m30_5, m30_6, m30_7, m30_8, m30_9, m30_10, m30_11, m30_12, m30_13, m30_14, m30_15, m30_16, m30_17, m30_18, m30_19, m30_20, m30_21, m30_22, m30_23, m30_24, m30_25, m30_26, m30_27, m30_28, m30_29, m30_30, m30_31, m30_32, \
+    m29_1, m29_2, m29_3, m29_4, m29_5, m29_6, m29_7, m29_8, m29_9, m29_10, m29_11, m29_12, m29_13, m29_14, m29_15, m29_16, m29_17, m29_18, m29_19, m29_20, m29_21, m29_22, m29_23, m29_24, m29_25, m29_26, m29_27, m29_28, m29_29, m29_30, m29_31, m29_32
+
+const vvc_dct2_64_odd_mat, dw matvec_mul_32_permute(dct2_64_odd_mat_permute( \
```

Comment:
vvc_itx_1d.asm:425: warning: single-line macro `matvec_mul_32_permute' exists, but not taking 1 parameter [-w+pp-macro-params-single]

Reply:
See https://github.com/ffvvc/FFmpeg/pull/114#discussion_r1269083847